### ISSUE TITLE
Issue with distributed windows exe

### ISSUE THREAD

**Issue by Mehgugs** [mdgriffith/elm-dev#22]
Hi! I downloaded elm-dev (by way of elm-prefab) and the package only downloads a single exe which does not work. I debugged this a little and found that there's some dynamic libraries missing on my system that the binary needs. I've included the output of Dependencies below. It's also worth noting that the official elm binary does not use these libraries.

![image](https://github.com/user-attachments/assets/885b40a6-94b2-4a7c-bf4a-aaad63d5af55)

**Comment by supermario** [comment]
Hey @Mehgugs – please try grab the DLL's from the Lamdera windows package, I think they should be the ones you're missing: https://dashboard.lamdera.app/docs/download

**Comment by Mehgugs** [comment]
Thank you @supermario! Would it be possible to have the npm dist for windows bring these along with it?

**Comment by supermario** [comment]
Yep it's possible, just needs to be done. @mdgriffith feel free to steal the Lamdera npm packaging script if you get to this before I do.
